Example/Kommunicate/LoginViewController.swift

```diff
@@ -19,14 +20,21 @@ class LoginViewController: UIViewController {
     @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
     @IBOutlet weak var loginAsVisitorButton: UIButton!
 
+    var isKeyboardVisible = false
```

Comment:
Can we add a comment here why this variable is needed? Since we are using notification for keyboard.